docs/guides/fine_tuning.md

```diff
@@ -137,7 +152,7 @@ Once your fine-tune is finished, you will be able to start making inference requ
 from llmengine import Completion
 
 response = Completion.create(
-    model_name="ft_abc123",
+    model_name="llama-7b:airlines:2023-07-17-08-30-45",
```

Comment:
Hmm I thought we changed this to `model` @squeakymouse @yunfeng-scale @phil-scale 

In any case, not a part of this PR